orion/tests/test_jira_viz_attachment.py

```
@@ -0,0 +1,163 @@
+                )
+
+            # Collect all attach_file calls
+            attach_calls = provider.attach_file.call_args_list
+
+            # BUG: PR 1111's viz is attached to PERF-200 (PR 2222's issue)
+            # and PR 2222's viz is attached to PERF-100 (PR 1111's issue)
+            pr1_viz = build_viz_output_file(base, "node-density", "pull_1111")
+            pr2_viz = build_viz_output_file(base, "node-density", "pull_2222")
+
+            # Each PR's viz should only attach to its own issue.
+            # Correct behavior: 2 calls total (1 per PR × 1 issue each)
```

Ruff (0.15.21)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)